### DISCUSSION TITLE
posts/polyclimate-global-projections/

### DISCUSSION THREAD

**Discussion by giscus** [polyclimate/polyclimate.org#5] [Announcements]
# posts/polyclimate-global-projections/

Carbon Budgets, Temperature Pathways and a proposal for Global Projections based on stylized scenarios.

https://polyclimate.org/posts/polyclimate-global-projections/

**Comment by mikapfl** [comment]
Nitpick:

> 2100, when, using average global life expectancy, everybody currently alive and their children won’t be around anymore.

Huh? People born today only need to become 77 to be alive in 2100! The future is closer than you think.
There is probably a non-zero chance that *you* are still alive in 2100 (a little bit of medical progress, healthy living, a stable climate - some people reach an age well in excess of 110 years today).

**Reply by rgieseke** [reply]
Sure, some will be around! 🙂
_Average global_ life expectancy is 73 though, according to WHO.

https://www.who.int/data/gho/data/indicators/indicator-details/GHO/life-expectancy-at-birth-(years)

I put it like this because I think "end of the century" warming is _far_ away for most people and these assessments rarely talk about the trajectory of getting there.

**Comment by rgieseke** [comment]
The link to the GitHub repo with the Notebooks has been fixed. https://github.com/polyclimate/polyclimate1

**Comment by rgieseke** [comment]
To further play with this idea I have created an interactive demo:

https://polyclimate.github.io/polyclimate1/demos/pathway-demo/persistence/
